lib/utils.ts

```
@@ -0,0 +1,311 @@
+  } catch {
+    return new MediaRecorder(stream);
+  }
+};
+
+export const getVideoDuration = (url: string): Promise<number | null> =>
+  new Promise((resolve) => {
+    const video = document.createElement("video");
+    video.preload = "metadata";
+    video.onloadedmetadata = () => {
+      const duration =
+        isFinite(video.duration) && video.duration > 0
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)